views/tables/ParticipantsTable/index.jsx

```diff
@@ -135,14 +136,18 @@ const ParticipantsTable = (props) => {
           />
         )
       case 'study':
+        const tooltipTitle = participant[property] in SITE_NAMES ? SITE_NAMES[participant[property]] : participant[property]
```

Comment:
I noticed we look at `participant[property]` a number of times in this block. What do you think about extracting that to a variable?

```suggestion
        const participantProperty = participant[property]
        const tooltipTitle = SITE_NAMES[participantProperty] || participantProperty
```

We can use `participantProperty` below in the Typography component as well.